docker/structs.go

```diff
@@ -0,0 +1,8 @@
+package docker
+
+import "github.com/docker/engine-api/types"
+
+// VolumeRequest is taken from
+// https://github.com/docker/engine-api/types/types.go
+type VolumeRequest types.VolumeCreateRequest
```

Comment:
I wanted a type mapping because docker's API is still extremely volatile, and not all structs from the volumes API are represented yet.

This gets compiled away so it should be performant. I asked her to put it in `docker/` but I think you have a good point about putting it in `api/` instead. @yuva29 do you have any objections to doing that?